juriscraper/opinions/united_states/state/colo.py

```diff
@@ -285,16 +282,22 @@ def _get_nature_of_suit(self):
     @staticmethod
     def cleanup_content(content):
         tree = html.fromstring(content)
```

Comment:
Does this really work this way? I mean, if you do `html.fromstring` then `tree.xpath` on a PDF, do you simply wind up with a falsy `core_element` value? I'd expect a crash, not just a failure to get data?